platform/platform-configuration/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -55,7 +55,7 @@
                 description="Turns on a system usage message, which is shown when the Search Application is opened"
                 name="Enable System Usage Message" id="systemUsageEnabled" required="true"
                 type="Boolean"
-                default=""
```

Comment:
Nice add on setting the default boolean values to false in the metatype files, something to look at in other PRs.